hangupsbot/plugins/telesync/core.py

```diff
@@ -557,6 +566,34 @@ def _on_membership_change(self, msg):
                 bot.memory.set_by_path(path_chat, 1)
                 type_ = 1
 
+                # check for chat restrictions and apply if necessary
+                restrict = get_sync_config_entry(bot,
+                                                 'telesync:' + msg.chat_id,
+                                                 'restrict_user')
+                rights = (NO_SENDING_RIGHTS if restrict == 'messages' else
+                          NO_MEDIA_RIGHTS if restrict == 'media' else
+                          NO_STICKER_RIGHTS if restrict == 'sticker' else
+                          NO_WEBPREVIEW_RIGHTS if restrict == 'websites' else
+                          NO_WEBPREVIEW_AND_STICKER_RIGHTS if restrict == 'sticker+websites'
+                          else None)
+                if rights and msg['chat']['type'] == 'supergroup':
+                    result = await self.restrictChatMember(msg.chat_id,
+                                                           changed_member.usr_id,
+                                                           **rights)
+
+                    if result is not True:
+                        mod_chat = self.config('mod_chat') if 'mod_chat' in self.config() else None
```

Comment:
What about an entry in `plugins.telesync.setup_config.default_config` for `mod_chat`?